docs/ml/_index.md

```diff
@@ -16,19 +16,55 @@ menuindent: true
 {{<imgproc src="/ml/training.png" class="alignright" resize="400x" declaredimensions=true alt="ML training">}}
 
 Machine Learning (ML) provides your machines with the ability to adjust its behavior based on models that recognize patterns or make predictions.
+
 Common use cases include:
 
-- Object detection and classification which enable machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object detection, which enables machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object classification, which enables machines to separate people, animals, plants, or other objects into predefined categories based on their characteristics, and to perform different actions based on the classes of objects.
 - Speech recognition, natural language processing, and speech synthesis, which enable machines to verbally communicate with us.
 
-However, your machine can make use of machine learning with nearly any kind of data.
+For other use cases, consider [creating custom functionality with a module](/registry/create/).
+
+Viam provides two services that enable machine learning capabilities: the [ML model](/ml/deploy/) service and the [Computer Vision](/ml/vision/) service.
+
+The ML model service deploys and runs a machine learning model, such as a TensorFlow or ONNX model, on your machine and makes its output accessible to other services.
+One service that is built on top of the ML model service is the [Computer Vision](/ml/vision/mlmodel/) `mlmodel` service which can detect or classify objects.
```

Comment:
```suggestion
The [Computer Vision](/ml/vision/mlmodel/) `mlmodel` service, which can detect or classify objects, is built to work with the inferences from an ML model service.
```

Reply:
@npentrel I think you suggested this line specifically to show the flexibility but the wording feels strange to me-- would this be ok substitute? Or too much info

Reply:
feel free to change!